user/serializers.py

```diff
@@ -22,3 +31,41 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        label=_("Email"),
+        write_only=True
+    )
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True
+    )
+    token = serializers.CharField(
+        label=_("Token"),
+        read_only=True
+    )
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(request=self.context.get("request"),
+                                username=email, password=password)
```

Comment:
In the `AuthTokenSerializer`, the `authenticate` function is called with `username=email`. Since the custom User model uses email as the `USERNAME_FIELD`, this is correct. However, ensure that this is consistent with the authentication backend being used.